parm/config/config.atmensanl

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash -x
+
+########## config.atmensanl ##########
+# configuration common to all atm analysis tasks
+
+echo "BEGIN: config.atmensanl"
+
+export OBS_YAML_DIR=${HOMEgfs}/sorc/gdas.cd/parm/atm/obs/config/
+export OBS_LIST=${HOMEgfs}/sorc/gdas.cd/parm/atm/obs/lists/gdas_prototype.yaml
+export ATMVARYAML=${HOMEgfs}/sorc/gdas.cd/parm/atm/variational/3dvar_dripcg.yaml
+export STATICB_TYPE="identity"
+export BERROR_YAML=${HOMEgfs}/sorc/gdas.cd/parm/atm/berror/staticb_${STATICB_TYPE}.yaml
+export FV3JEDI_FIX=${HOMEgfs}/fix/gdas
+export R2D2_OBS_DB='ufsda_test'
+export R2D2_OBS_DUMP='oper_gdas'
+export R2D2_OBS_SRC='ncdiag'
+export R2D2_BC_SRC='gsi'
+export R2D2_BC_DUMP='oper_gdas'
+export R2D2_ARCH_DB='local'
+export INTERP_METHOD='barycentric'
+
+export layout_x=1
+export layout_y=1
+
+export io_layout_x=1
+export io_layout_y=1
+
+export JEDIVAREXE=${HOMEgfs}/exec/fv3jedi_var.x
+export CRTM_VER="2.3.0"
```

Comment:
Resolved as noted above.  Done at 6ae4c3a8.

Reply:
@WalterKolczynski-NOAA 

Unfortunately this is not resolved.   When we run UFS DA, workflow jobs execute `ush/load_ufsda_modules.sh`.   This loads `sorc/gdas.cd/modulefiles/GDAS/$machine.lua` where `machine` is `hera` or `orion` depending on the platform on which you are running.     The following crtm modules are available on Hera and Orion.
```
     Versions:
        crtm/v2.3-jedi.4
        crtm/v2.4_jedi
        crtm/2.3.0
        crtm/2.4.0
```
Of these, only the Orion `crtm/2.4.0` defines `crtm_VERSION`.    None of the Hera crtm modules define `crtm_VERSION`.   We can not assume `crtm_VERSION` is defined when running UFS DA.    

The Hera and Orion modulefiles in `gdas.cd/modulefiles/GDAS` do not load a crtm module.  Instead, `CRTM_FIX` is defined.    `CRTM_VER` is specified in UFS DA config files to identify the specific version of `CRTM_FIX` to use.    Currently the UFS DA `CRTM_FIX` only contains coefficients for `2.3.0` and `2.3.0_jedi`.    

For the time being we need to set `CRTM_VER` in UFS DA config files.     Hopefully future updates to UFS DA crtm modules will define `crtm_VERSION` on all supported machines.

Tagging @CoryMartin-NOAA for awareness.

Reply:
Since I already updated `aero_crtm_coeff.yaml` and `atm_crtm_coeff.yaml` to `crtm_VERSION`, I added `export crtm_VERSION=2.3.0` to `config.atmanl` as a test.   `gdasatmanlinit` was submitted.   It successfully ran to completion with the correct CRTM coefficients staged.   This isn't an ideal solution, but it works.

Reply:
Added `crtm_VERSION` to `config.atmanl` and `config.atmensanl` at c207ff66.